src/options/base.js

```diff
@@ -8,6 +8,7 @@ const elements = {
   saveFiltersButton: document.getElementById('save-filters'),
   accordionItems: document.querySelectorAll('.accordion .accordion-link'),
   enableSearchStorageCheckbox: document.getElementById('enable-search-storage-checkbox'),
+  enableSearchClearConfirm: document.getElementById('enable-search-clear-confirmation'),
```

Comment:
Let's follow the convention here and name the variable `enableSearchClearConfirmCheckbox`.

Reply:
I initially named it `enableSearchClearConfirmCheckbox` but then it is showing linter error.
The line exceed 100 char length.

Reply:
Oh, then you can span it to multiple lines. 
like this:
``` javascript
  enableSearchClearConfirm: document.getElementById(
    "enable-search-clear-confirmation",
  ),
```

Reply:
Ok. Thank you for the help.